clang/test/CodeGenOpenCL/sampled_image_cuda.cl

```diff
@@ -0,0 +1,15 @@
+// RUN: %clang_cc1 %s -triple nvptx64-nvidia-cuda -O0 -emit-llvm -o - | FileCheck %s
```

Comment:
I'm not sure that CodeGenOpenCL test is a right place for such test...

Reply:
Sorry, I don't understand how you addressed @Fznamznon's question.

Reply:
> Sorry, I don't understand how you addressed @Fznamznon's question.

No worries. It was only marked WIP because it wasn't passing the clang format check. I've resolved that now and removed the WIP status.

Reply:
> I'm not sure that CodeGenOpenCL test is a right place for such test...

I think @Naghasan can provide some insight into this.